python/test/test_units.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+
+from dataset import *
+import numpy as np
+
+class TestUnits(unittest.TestCase):
+    def test_create_unit(self):
+        u = units.angstrom
+        self.assertEqual(repr(u), "AA")
+
+    def test_variable_unit(self):
+        var1 = Variable(Data.Value, [Dim.X], np.arange(4))
+        var2 = Variable(Coord.X, [Dim.X], np.arange(4))
+        self.assertEqual(repr(var1.unit), "dimensionless")
+        self.assertEqual(repr(var2.unit), "m")
```

Comment:
Can you also do `self.assertEqual(var2.unit, units::m)`? If not, is comparison missing from the `Unit` exports?

I'd put a tests of `repr` as a separate test and use normal comparison wherever possible.